docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,91 @@
+6. Documents are protected from unauthorized changes and any deletion or removal
+7. Documents are protected from unauthorized access
+8. The unintended use of obsolete documents is prevented, and suitable identification is applied to them if they are retained for any purpose
+9. One paper or electronic copy of each obsolete controlled document is retained for a specified time period or in accordance with applicable specified requirements
+
+## Candidates for a Bioinformatician friendly QMS
+
+_Bioinformaticians_ create workflows collaboratively on a computer within a git environment. Git is a distributed version control system that tracks changes in any set of computer files. This can be used to track the changes of the code itself, but also in its documentation, which can describe the Standard Operating Procedures (SOP) to use it. Git comes in with a set of features allowing to track issues and their resolution, new release of a workflow and its newest features. As the workflow often get upgraded and updated, their documentation, as well, is susceptible to change. Which is why many chose _Bioinformaticians_ their own form of QMS. Examples of those documentations systems are:
 
 1. Github pages
-2. Gitlab
-3. Confluence
\ No newline at end of file
+2. Gitlab pages
+3. Confluence
+
+Many of those tools, if used in a specific way, can allow _Bioinformaticians_ to work in a comfortable environement, while complying to the ISO norms requirements. They are of course multiple ways to implement such a system, and as long as it is documented how your system complies to the ISO requirements, all is well. If you are looking for some inspirations, in the next paragraph we will describe an example on how to use one this tool to follow the ISO requirements for both controlling the workflows/software you develop, and their associated documentation/SOP.
+
+## Example case : The usage of GitLab
+
+### Central gathering of all SOP on Gitlab pages
+
+Using gitlab pages and mkdocs, you can easily create a well structured _documentation_ to store all your SOPs . Here are some guideline to illustrates the point of attention to comply with ISO-15189 :
+
+1. Each SOP becomes a markdown file. Its name will include a number for easy and unique identification of the document. The numbering can also be used to classify each SOP into categories. _Example_ :
+   
+- 10 - Data Access
+    - 11 - ExternalSequencers -> 11_externalsequencers.md
+    - 12 - InternalSequencers -> 12_internalsequencers.md
+- 20 - Data Processing
+    - 21 - WES -> 21_wes_flow.md
+    - 22 - RNAseq -> 22_rnaseq_flow.md
+- 30 - Data sharing
+    - 31 - Cluster -> 31_share_cluster_user.md
+    - 32 - External -> 32_share_external_user.md      
+
+ Here we seperate 3 flows in categories : accessing data, processing data and sharing data, ✨ _allowing smoothe and logical navigation within your documentation_ ✨  (sorry guys, I can't help it, I need my _beautiful_ sentences)
```

Comment:
I like the emojis :)
but `smooth' is without an `e' at the end.

Reply:
I used intuitive instead :p That is the word I was looking for all those months. Found it today after second coffee